pkg/thing/controllers/thing.go

```diff
@@ -66,8 +78,24 @@ func (mc *ThingController) UpdateSchema(body []byte, authorizationHeader string)
 }
 
 // ListDevices handles the list devices request and execute its use case
-func (mc *ThingController) ListDevices(authorization, replyTo, corrID string) error {
+func (mc *thingController) ListDevices(authorization, replyTo, corrID string) error {
 	mc.logger.Info("list devices command received")
+	if replyTo == "" {
+		sendErr := mc.sender.SendListResponse([]*entities.Thing{}, replyTo, corrID, interactors.ErrReplyToNotProvided)
+		if sendErr != nil {
+			return fmt.Errorf("error sending response: %v: %w", interactors.ErrReplyToNotProvided, sendErr)
+		}
+		return interactors.ErrReplyToNotProvided
+	}
+
+	if corrID == "" {
```

Comment:
As discussed with @vitbaq [here](https://github.com/CESARBR/knot-babeltower/pull/79#discussion_r433866643), I think correlationId should be optional.